Testing/ORKTest/ORKTest/TaskFactory.h

```diff
@@ -0,0 +1,93 @@
+//
+//  TaskFactory.h
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#define DefineStringKey(x) static NSString *const x = @#x
+
+DefineStringKey(ConsentTaskIdentifier);
```

Comment:
I guess I kind of figured that now, while you were doing the initial move to a new file, and while there are only three outstanding PR's that touch `ORKTest` would be the time to do it.